prow/scripts/cluster-integration/kyma-upgrade-skr-aws-kyma-to-kyma2.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+#Description: Kyma Integration plan on SKR AWS. This scripts implements a pipeline that consists of many steps. The purpose is to trigger the ci-skr-kyma-to-kyma2-upgrade fast-integration test in Kyma repository
+#Expected common vars:
+#
+#
+#Please look in each provider script for provider specific requirements
+
+
+
+set -o errexit
+set -o pipefail
+
+function prereq() {
+    export TEST_INFRA_SOURCES_DIR="${KYMA_PROJECT_DIR}/test-infra"
+    # shellcheck source=prow/scripts/lib/utils.sh
+    source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/utils.sh"
+    # shellcheck source=prow/scripts/lib/kyma.sh
+    source "${TEST_INFRA_SOURCES_DIR}/prow/scripts/lib/kyma.sh"
+
+    # All provides require these values, each of them may check for additional variables
+    requiredVars=(
+        KYMA_VERSION
+    )
+    utils::check_required_vars "${requiredVars[@]}"
+}
+
+#Used to detect errors for logging purposes
+ERROR_LOGGING_GUARD="true"
+export ERROR_LOGGING_GUARD
+
+prereq
+
+log::info "### Starting pipeline"
+
+# Fetch latest Kyma2 release
+kyma::get_last_release_version -t "${BOT_GITHUB_TOKEN}"
+export KYMA_UPGRADE_VERSION="${kyma_get_last_release_version_return_version:?}"
+log::info "### Reading release version from RELEASE_VERSION file, got: ${KYMA_UPGRADE_VERSION}"
```

Comment:
Just try to avoid misinterpretation, The release version actually read from github release API by the definition of `kyma_get_last_release_version_return_version` [function](https://github.com/kyma-project/test-infra/blob/main/prow/scripts/lib/kyma.sh#L92).